### PR TITLE
Added common issues & solutions in operator install

### PR DESCRIPTION
**Pull request by kirtanchandak**
- added the docker root privilege in the new section common issues and solutions 

Related issues
https://github.com/vitessio/website/issues/1738

### CODE REVIEW

File: content/en/docs/18.0/get-started/operator.md [hunk 1]
```diff
@@ -161,6 +161,12 @@ The schema has been simplified to include only those fields that are significant
 * The `customer` table has a `customer_id` that has an `auto_increment`. A typical customer table would have a lot more columns, and sometimes additional detail tables.
 * The `corder` table (named so because `order` is an SQL reserved word) has an `order_id` auto-increment column. It also has foreign keys into `customer(customer_id)` and `product(sku)`.
 
+## Common Issues and Solutions
+
```
Comment (deepthi) [inline content/en/docs/18.0/get-started/operator.md:165]: Can you include the error message here?

File: content/en/docs/18.0/get-started/operator.md [hunk 2]
```diff
@@ -161,6 +161,12 @@ The schema has been simplified to include only those fields that are significant
...
+* The `docker` driver should not be used with root privileges.
+
+  Solution - Create a new user and it to your [docker group](https://docs.docker.com/engine/install/linux-postinstall).
```
Comment (deepthi) [inline content/en/docs/18.0/get-started/operator.md]:
```suggestion
  Solution - Create a new user and add it to your [docker group](https://docs.docker.com/engine/install/linux-postinstall).
```

File: content/en/docs/18.0/get-started/operator.md (hunk 1 above)
Comment (kirtanchandak) [reply content/en/docs/18.0/get-started/operator.md:165]: like the actual error message? which lines from here?

image

Comment (deepthi) [reply content/en/docs/18.0/get-started/operator.md:165]: The sentence starting with "The "docker" driver ..."

Comment (kirtanchandak) [reply content/en/docs/18.0/get-started/operator.md:165]: hey, I've updated!

File: content/en/docs/18.0/get-started/operator.md (hunk 2 above)
Comment (kirtanchandak) [reply content/en/docs/18.0/get-started/operator.md]: added code block for the error message as discussed

Comment (deepthi) [reply content/en/docs/18.0/get-started/operator.md]: Looks like the code block was added only in 19.0 docs. The other versions also need to be fixed to match this.

Comment (kirtanchandak) [reply content/en/docs/18.0/get-started/operator.md]: made it for all, ready to merge :)

**Review by deepthi — APPROVED** [review]

**Pull request merged**